libs/platform/experience/src/services/layout/plugins/styles/misc/misc-layout.model.ts

```diff
@@ -0,0 +1,110 @@
+import { LayoutAlign } from '../../../layout.model';
+
+export {};
+
+declare global {
+  export interface LayoutStylesProperties {
+    /**
+     * The column count is based on a calculated count by design tokens, which can be specified
+     * by design tokens. This results in a column system for grid and carousels that uses 4, 2
+     * or 1 columns for large, medium and small screens.
+     *
+     * While the column system can be configured globally by design tokens, the `columnCount` allows
+     * for an individual override per layout.
+     *
+     * It is also worth mentioning that nested layouts will inherit the column size from the parent
+     * layout, by taken the parent item _span_ count, which default to `1`. This results in a
+     * balanced layout, where nested layouts will reflect the parent column setup.
+     */
+    columnCount?: number;
```

Comment:
`columnCount`, `align`, `justify`, `span`, `colSpan`, `rowSpan`, `gridRow`, `gridColumn` and `gap` are all related to layout. Used by many layouts. Could we move them to a layout style plugin? Or can we use them in different layout plugins (e.g. column, carousel and grid)? In the latter case, they would be used multiple times.